lib/ngx/errlog.lua

```diff
@@ -87,4 +89,10 @@ function _M.get_logs(max, logs)
 end
 
 
+function _M.get_sys_filtering_level()
+    local r = getfenv(0).__ngx_req
+    return C.ngx_http_lua_ffi_errlog_get_sys_filtering_level(r)
```

Comment:
We should not return the cdata value directly without converting it to a number via the `tonumber()` builtin.